### DISCUSSION TITLE
In extension list link to the website

### DISCUSSION THREAD

**Discussion by bobonov** [mlocati/docker-php-extension-installer#791] [Ideas]
It would be useful to add a link to the related extension website.
Sometimes I use this list also to check what's new in php extension world, but for some of the extensions it is hard to find a reference to check what it does.
I know that when you decide to use an extension is because you already know it, but from this list you can get also "inspiration" to improve your work.

**Comment by mlocati** [comment]
Most of the packages are available either from the [PECL website](https://pecl.php.net/) or from the [main PHP source code](https://github.com/php/php-src/tree/master/ext).

The only exceptions are:
- [composer](https://getcomposer.org/)
- [blackfire](https://www.blackfire.io/)
- [cassandra](https://github.com/nano-interactive/ext-cassandra)
- [geos](https://git.osgeo.org/gitea/geos/php-geos)
- [ioncube_loader](https://www.ioncube.com/loaders.php)
- [lz4](https://github.com/kjdev/php-ext-lz4)
- [relay](https://relay.so/)
- [snappy](https://github.com/kjdev/php-ext-snappy)
- [snuffleupagus](https://github.com/jvoisin/snuffleupagus)
- [sourceguardian](https://www.sourceguardian.com/)
- [spx](https://github.com/NoiseByNorthwest/php-spx)
- [zmq](https://github.com/zeromq/php-zmq)
